atox/src/main/res/layout/fragment_settings.xml

```diff
@@ -171,6 +171,27 @@
                                 app:layout_constraintEnd_toEndOf="parent"
                                 app:layout_constraintTop_toTopOf="parent"/>
                     </androidx.constraintlayout.widget.ConstraintLayout>
+                    <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
Isn't the commit message set on merge, anyway?

Reply:
No, that would mean GitHub would be ruining your GPG-signature on the commit since the message is part of what's signed. :P

I can do it manually, but that's me checking the branch out locally, amending the commit message, force-pushing your branch, and then merging once CI has passed again.

Reply:
Okay, I'll change it once I'm done with the other branch.